eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Generator/Utxo.hs

```diff
@@ -63,7 +63,7 @@ import Control.State.Transition
 import qualified Data.ByteString.Lazy as BSL
 import qualified Data.Either as Either (partitionEithers)
 import qualified Data.IntSet as IntSet
-import Data.List (foldl')
+import Data.List as List (foldl')
```

Comment:
For consistency sake again, we should start using `Foldable`: 
```suggestion
import Data.Foldable as F (foldl')
```

Reply:
Done